components/transaction/internal/adapters/postgres/transaction/transaction.go

```
@@ -510,43 +511,49 @@ func (cti *CreateTransactionInput) BuildTransaction() *pkgTransaction.Transactio
-				Amount: &pkgTransaction.Amount{
-					Asset: op.AssetCode,
-					Value: *op.Amount.Value,
-				},
+			froms = append(froms, pkgTransaction.FromTo{
+				IsFrom:          true,
+				AccountAlias:    op.AccountAlias,
+				Amount:          &pkgTransaction.Amount{Asset: op.AssetCode, Value: *op.Amount.Value},
 				Description:     op.Description,
 				ChartOfAccounts: op.ChartOfAccounts,
 				Metadata:        op.Metadata,
 				Route:           op.Route,
```

golangci-lint (2.11.3)

[error] SA1019: op.Route is deprecated: legacy route identifier, use routeId instead. Contains the same operation route UUID as routeId but stored as a free-form string for backwards compatibility. example: 00000000-0000-0000-0000-000000000000 maxLength: 250 deprecated: true 

(staticcheck)